html-templates/sbg/progress/section-term-reports/sbg.body.tpl

```diff
@@ -0,0 +1,99 @@
+{$gradeLabels = array(
+    "N/A" = "Standard not Applicable during the Semester"
+    ,1 = "Not currently meeting expectations"
+    ,2 = "Approaching expectations"
+    ,3 = "Meeting expectations"
+    ,4 = "Exceeding expectations"
+)}
+
+{template standards Report gradeLabels}
```

Comment:
there's no longer any reason for this to be broken out as a subtemplate, the content might as well be in-context where it's used and indented accordingly